controller/share_manager_controller.go

```diff
@@ -316,6 +366,33 @@ func (c *ShareManagerController) syncShareManager(key string) (err error) {
 		return c.ds.RemoveFinalizerForShareManager(sm)
 	}
 
+	isStale, leaseHolder, err := c.isShareManagerPodStale(sm)
+	if err != nil {
+		return err
+	}
+	if isStale {
+		isDelinquent, _, err := c.ds.IsRWXVolumeInDelinquent(sm.Name)
```

Comment:
NIT: Function name reads strangely. After all the changes, it seems that this could just be:

```suggestion
		isDelinquent, _, err := c.ds.IsRWXVolumeDelinquent(sm.Name)
```

Reply:
Agreed, although that can't be a simple commit suggestion from here, because the interface would need to change in datastore, too.

Reply:
Thank you. Fixed